### ISSUE TITLE
[Bug]: bounty and wanted level

### ISSUE THREAD

**Issue by ChaT0n** [Yimura/YimMenu#1544]
### Describe the bug

When I choose a player in the session where I am, and I try to put a bounty on him after choosing the amount, it doesn't work ( in both anonymous or not anonymous )
Also when I try to chnage the wanted level of the player, he isn't affected
There were no other admins in the session blocking those actions

### Steps To Reproduce

1. Go to the list of players in the session when opening the menu
2. Choose a player and go to toxic
3. Try to change the wanted level or put a bounty ( anonymous or normally)

### Expected Behavior

changing the wanted level of the choosen player from the session, and putting a bounty on the choosen player

### Screenshots and/or Logs

```
<paste your logs here>
```

### Store

Steam

### Additional context

tried it witha friend in a closed session and still din't work

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by PinkyPromises** [comment]
Still hasn't been fixed

**Comment by lonelybud** [comment]
For me the set bounty is not working on another players. Using the latest dll build.

**Comment by ghost** [comment]
Strange, but now the bounty works again. Probably in some PR they fixed the problem or it fixed itself. Check with yourself again.

**Comment by ghost** [comment]
Yes, I tried this the other day to troll my friend, but it didn't work (my friend doesn't use mods).

715af52 I tried on this version
